delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/utils/OwnerPermissionsScopeInitialization.scala

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.bluebrain.nexus.delta.service.utils
+
+import ch.epfl.bluebrain.nexus.delta.kernel.syntax._
+import ch.epfl.bluebrain.nexus.delta.sdk._
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ScopeInitializationFailed
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.{Acl, AclAddress, AclRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.{Caller, ServiceAccount}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.Organization
+import ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.Permission
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.Project
+import com.typesafe.scalalogging.Logger
+import monix.bio.{IO, UIO}
+
+/**
+  * The default creation of ACLs for newly created organizations and projects. It performs a noop if
+  * executed during a migration.
+  *
+  * @param acls             the acls module
+  * @param ownerPermissions the collection of permissions to be granted to the owner (creator)
+  * @param serviceAccount   the subject that will be recorded when performing the initialization
+  */
+class OwnerPermissionsScopeInitialization(acls: Acls, ownerPermissions: Set[Permission], serviceAccount: ServiceAccount)
+    extends ScopeInitialization {
+
+  private val logger: Logger          = Logger[OwnerPermissionsScopeInitialization]
+  implicit private val caller: Caller = serviceAccount.caller
+
+  override def onOrganizationCreation(
+      organization: Organization,
+      subject: Subject
+  ): IO[ScopeInitializationFailed, Unit] =
+    if (MigrationState.isRunning) IO.unit
```

Comment:
If we had this `ScopeInitializationRunner` (or whatever it may be called), we could also disable this operations in a single point

Reply:
Again migrations code creeping in production code....